packages/p2p/src/components/buy-sell/buy-sell-table/__tests__/buy-sell-table-no-ads.spec.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import userEvent from '@testing-library/user-event';
+import { render, screen, waitFor } from '@testing-library/react';
+import { useStores } from 'Stores/index';
+import BuySellTableNoAds from '../buy-sell-table-no-ads';
+
+const mock_store: DeepPartial<ReturnType<typeof useStores>> = {
+    buy_sell_store: {
+        is_buy: true,
+        local_currencies: [
+            {
+                text: 'IDR',
+                is_default: true,
+            },
+            {
+                text: 'INR',
+                is_default: false,
+            },
+        ],
+        selected_local_currency: 'IDR',
+        setCreateSellAdFromNoAds: jest.fn(),
+    },
+    general_store: {
+        handleTabClick: jest.fn(),
+        is_barred: false,
+    },
+    my_ads_store: {
+        setShowAdForm: jest.fn(),
+    },
+};
+
+jest.mock('Stores', () => ({
+    ...jest.requireActual('Stores'),
+    useStores: jest.fn(() => mock_store),
+}));
+
+describe('BuySellTableNoAds', () => {
+    it('shoulde render the component if there is no ad for the local currency selected', async () => {
```

Comment:
`it('shoulde render the component if there is no ad for the local currency selected', async () => {`